### PR TITLE
Digital Credentials: test static methods from Credential interface

### PR DESCRIPTION
**Pull request by marcoscaceres**
cc @pkotwicz @samuelgoto

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@pkotwicz, can I add you as reviewer to the META.yaml file associated with the digital credentials directory?

File: digital-credentials/digital-credentials-static-methods.tentative.https.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<script src="/resources/testharnessreport.js"></script>
+
+<body></body>
+<script type="module">
+  /**
+   * @see https://www.w3.org/TR/credential-management/#dom-credential-isconditionalmediationavailable
+   */
+  promise_test(async (t) => {
+    const proto = Object.getPrototypeOf(navigator.identity);
+    const result = await  await proto.isConditionalMediationAvailable();
```
Comment (marcoscaceres) [inline digital-credentials/digital-credentials-static-methods.tentative.https.html:14]:
```suggestion
    const result = await proto.isConditionalMediationAvailable();
```

File: digital-credentials/digital-credentials-static-methods.tentative.https.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<body></body>
+<script type="module">
+  /**
+   * @see https://www.w3.org/TR/credential-management/#dom-credential-isconditionalmediationavailable
+   */
+  promise_test(async (t) => {
+    const proto = Object.getPrototypeOf(navigator.identity);
+    const result = await proto.isConditionalMediationAvailable();
+    assert_equals(result, false);
+  }, "check default isConditionalMediationAvailable() behavior, inherited from Credential interface.");
```
Comment (marcoscaceres) [inline digital-credentials/digital-credentials-static-methods.tentative.https.html:16]:
```suggestion
  }, "check default isConditionalMediationAvailable() behavior from the Credential interface, but used in the context of digital credentials.");
```

File: digital-credentials/digital-credentials-static-methods.tentative.https.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  /**
+   * @see https://www.w3.org/TR/credential-management/#dom-credential-willrequestconditionalcreation
+   */
+  promise_test(async (t) => {
+    const proto = Object.getPrototypeOf(navigator.identity);
+    const result = await proto.willRequestConditionalCreation();
+    assert_equals(result, undefined);
+  }, "check default willRequestConditionalCreation() behavior, inherited from Credential interface");
```
Comment (marcoscaceres) [inline digital-credentials/digital-credentials-static-methods.tentative.https.html:25]:
```suggestion
  }, "check default willRequestConditionalCreation() behavior from the Credential interface, but used in the context of digital credentials.");
```

**Review by samuelgoto — APPROVED** [review]
LGTM

**Review by annevk — APPROVED** [review]
rubber-stamp based on Sam's review.

**Pull request merged**